src/main/java/com/pnu/ordermanagementapp/client/ClientServiceImpl.java

```diff
@@ -4,33 +4,44 @@
 import com.pnu.ordermanagementapp.model.Client;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Service;
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 @Service
 public class ClientServiceImpl implements ClientService {
 
-    private ClientRepository repository;
+    private static final int PAGE_SIZE = 10;
+
+    private static final Sort SORT = Sort.by("active").descending().and(Sort.by("id").descending());
+
+    private ClientRepository clientRepository;
 
     @Autowired
-    public ClientServiceImpl(ClientRepository repository) {
-        this.repository = repository;
+    public ClientServiceImpl(ClientRepository clientRepository) {
+        this.clientRepository = clientRepository;
     }
 
     @Override
     public List<Client> findAll(Long userId) {
```

Comment:
If you change the behaviour of some method, try to look at the usages, because it may crash something.
As far as I remember this method is used in OrderController, replace usages of this method by findAllByActivity with flag `true`.
Also, check if `findAll` will be used somewhere after it, if no - simply remove this method at all.

And just for readability improvement, I'd avoid passing boolean flags to method and instead of `findAllByActivity` wound create two seperate methods `findAllActive` and `findAllInactive` that would just call out to `findAllByActivity` with appropriate flag. Method `findAllByActivity` should be private in such case.